Updates/0237_rp_klaven_mortwake.sql

```diff
@@ -0,0 +1,199 @@
+-- Update RP Event for Klaven Mortwake
+-- Update spawn and Waypoints from Klaven Mortwake
+UPDATE `creature` SET `position_x`='-11127.023', `position_y`='547.55206', `position_z`='70.51061', `orientation`='0.12217' WHERE (`guid`='66996');
+
+SET @PATH := 7053;
+DELETE FROM creature_movement_template WHERE `Entry` = @PATH;
+INSERT INTO `creature_movement_template` (`Entry`, `PathId`, `Point`, `PositionX`, `PositionY`, `PositionZ`, `Orientation`, `WaitTime`, `ScriptId`) VALUES 
+(@PATH,0,1,-11127.023,547.55206,70.51061,100,8000,0),
+(@PATH,0,2,-11126.639,551.8031,70.433266,100,26000,705301),
+(@PATH,0,3,-11132.797,555.2593,70.42202,100, 0,0),
+(@PATH,0,4,-11136.433,557.4096,70.43819,100, 0, 0),
+(@PATH,0,5,-11141.278,558.67175,70.41653,100, 0, 0),
+(@PATH,0,6,-11145.746,558.1757,70.379524,100, 0, 0),
+(@PATH,0,7,-11151.536,555.5678,70.38759,100, 0, 0),
+(@PATH,0,8,-11156.048,547.96124,70.448845,100, 0, 0),
+(@PATH,0,9,-11158.312,543.91876,70.410866,100, 0, 0),
+(@PATH,0,10,-11163.021,542.7415,70.51954,100,8000,705302),
+(@PATH,0,11,-11161.916,536.54095,70.433266,100, 0, 0),
+(@PATH,0,12,-11161.916,536.54095,70.433266,3.36848,6000,705303),
+(@PATH,0,13,-11155.41,536.8843,70.42894,100, 0, 0),
+(@PATH,0,14,-11153.562,533.9012,70.42189,100, 0, 0),
+(@PATH,0,15,-11146.808,528.71625,70.477196,100, 0, 0),
+(@PATH,0,16,-11139.436,528.66187,70.50565,100, 0, 0),
+(@PATH,0,17,-11134.32,532.9369,70.562805,100, 0, 0),
+(@PATH,0,18,-11131.548,536.63214,70.47361,100, 0, 0),
+(@PATH,0,19,-11129.091,540.075,70.47177,100, 0, 0),
+(@PATH,0,20,-11128.235,543.1552,70.45672,100, 4000, 705304),
```

Comment:
Why do we need to switch path?

Reply:
He has something like an intro before pathing from x-y all the time thats why we switch path here